### PR TITLE
Tweak a couple of small things about the build

### PR DESCRIPTION
**Pull request by alexturek**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+6.2.2
```
Comment (nevir) [inline .nvmrc]: Let's have `circle.yml` read from this (https://github.com/convoyinc/dapper/blob/master/circle.yml#L5)

File: tsconfig.json [hunk 2]
```diff
@@ -13,5 +13,12 @@
     "outDir": "./dist/",
     "pretty": true,
     "target": "es5"
-  }
+  },
+  "include": [
+    "src/*",
+    "typings/*",
```
Comment (nevir) [inline tsconfig.json]: `typings/**` and drop the rest

File: tsconfig.json [hunk 3]
```diff
@@ -13,5 +13,12 @@
     "outDir": "./dist/",
     "pretty": true,
     "target": "es5"
-  }
+  },
+  "include": [
+    "src/*",
```
Comment (nevir) [inline tsconfig.json]: `src/**`

Comment (alexturek) [inline tsconfig.json]: Nope, Typescript 2.0 hates `folder/**`.
```
> ts2gql@1.0.1 compile /Users/aturek/greypoint/ts2gql-3
> ./scripts/compile

error TS5010: File specification cannot end in a recursive directory wildcard ('**'): 'src/**'.
```

File: tsconfig.json (hunk 2 above)
Comment (alexturek) [inline tsconfig.json]: See above comment

File: .nvmrc (hunk 1 above)
Comment (alexturek) [inline .nvmrc]: Done!

**Pull request merged**